masonry/src/widget/sized_box.rs

```diff
@@ -476,6 +505,19 @@ mod tests {
         assert_render_snapshot!(harness, "label_box_with_size");
     }
 
+    #[test]
+    fn label_box_with_padding() {
+        let widget = SizedBox::new(Label::new("hello"))
+            .border(Color::BLUE, 5.0)
+            .rounded(5.0)
+            .padding((60., 40.));
+
+        let mut harness = TestHarness::create(widget);
+
+        assert_debug_snapshot!(harness.root_widget());
```

Comment:
I'm not sure these debug snapshot tests are needed here.

Reply:
Should I remove them? I just copied them from the other test cases in the file.

Reply:
We can leave them from consistency. @PoignardAzur what are these snapshots actually testing?